docs/2_step_by_step/2_3_login_to_jina_ecosystem.md

```diff
@@ -1 +1,27 @@
-# Login to Jina ecosystem
\ No newline at end of file
+# Login to Jina ecosystem
+
+Since Finetuner leverages cloud resources for Fine-tuning.
+Before starting a fine-tuning job,
+it is required to login and obtain a token from Jina.
```

Comment:
Since Finetuner leverages cloud resources for fine-tuning, you are required to login and obtain a token from Jina before starting a fine-tuning job.